common/src/asset.rs

```diff
@@ -159,10 +210,26 @@ impl<T: AssetClass> Display for FungibleAsset<T> {
     }
 }
 
+impl<T: AssetClass> FromStr for FungibleAsset<T> {
+    type Err = near_account_id::ParseAccountError;
```

Comment:
```suggestion
    type Err = <AccountId as FromStr>::Err;
```

Eliminates the need to introduce the `near-account-id` dependency.

Reply:
It already is a dependency, but would remove the dependency for external crates, so yes good chance.